packages/storage/package.json

```diff
@@ -15,6 +15,7 @@
     "lint:fix": "eslint --fix -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
     "build": "rollup -c rollup.config.compat.js",
     "build:exp": "rollup -c rollup.config.exp.js",
+    "build:exp:release": "yarn build:exp && yarn api-report",
```

Comment:
Don't we always want to build the API report to ensure that we don't accidentally change the public API?

Reply:
Probably, changed.